frontend/components/SetupPage/Create/SetupCreateSafe.tsx

```diff
@@ -15,10 +15,14 @@ import { delayInSeconds } from '@/utils/delay';
 
 const { Text } = Typography;
 
-const capitalizedMiddlewareChainNames = {
-  [+MiddlewareChain.ETHEREUM]: 'Ethereum',
-  [+MiddlewareChain.BASE]: 'Base',
-  [+MiddlewareChain.OPTIMISM]: 'Optimism',
+const capitalizedMiddlewareChainNames: { [key in MiddlewareChain]: string } = {
```

Comment:
can't we use `EvmChainName(asMiddlewareChain(chain))` and remove this?

Reply:
Probably, I don't like this implementation but seems added from the merge